
std_msgs/Header header
float64 gt_mass
float64 gt_Jxy
float64 gt_com_z